docs/contributing: add info for replicated's about gcp/codeserver env (#3714)
replicatedhq/kURL: CONTRIBUTING.md

### Using GCP

> **If you are a Replicated team member then:** ensure that you follow up the doc [Getting Started with your own CodeServer][code-server].

Reviewer: ```suggestion
**If you are a Replicated team member then:** ensure that you have followed [Getting Started with your own CodeServer](https://github.com/replicatedhq/codeserver/blob/main/docs/first-time.md).
```

Author: Use alias make easier we keep the docs maintained since all links would be at the bottom

Author: However, it is not a big deal eaither.

### QEMU on MacOS

> [code-server]: https://github.com/replicatedhq/codeserver/blob/main/docs/first-time.md

Reviewer: ```suggestion
```
